### PR TITLE
Fixes F#53

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: sr_object_tracking/launch/tracking.launch [hunk 1]
```diff
@@ -15,9 +15,7 @@
   </group>
 
   <group unless="$(arg kinect)">
-    <node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
-        <remap from="/image_raw" to="/camera/rgb/image_color"/>
-    </node>
+    <include file="$(find autopic)/launch/camera.launch" />
```
Comment (ugocupcic) [inline sr_object_tracking/launch/tracking.launch]: No! You can't have this public launch file dependent on a private project :)

The way to do it is to add a no camera option to this launch file and then call it from autopic's launch file.

File: sr_object_tracking/launch/tracking_camera.launch [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <launch>
```
Comment (ugocupcic) [inline sr_object_tracking/launch/tracking_camera.launch]: so this should use the newly created launch file?

Comment (ghost) [inline sr_object_tracking/launch/tracking_camera.launch]: Well, no.. I separated my launch file in two : one which uses the camera (this one) and the other which doesn't (tracking_no_camera). It's the last one that should be used with autopic then. 
Wasn't it the idea ?

File: sr_object_tracking/launch/tracking.launch [hunk 3]
```diff
@@ -31,9 +23,15 @@
...
-  </node>
+  <group if="$(arg display)">
+    <node name="viewer" pkg="sr_gui_servoing" type="displaying.py" >
+      <param name="saturation" value="$(arg saturation_min)" />
+      <param name="color" value="$(arg color)" />
+    </node>
+  </group>
+
+  <group if="$(arg camera)">
+    <include file="$(find sr_object_tracking)/launch/camera.launch"/>
```
Comment (ugocupcic) [inline sr_object_tracking/launch/tracking.launch]: you'll need to pass a kinect argument through as well (set it in tracking.launch, pass it to camera.launch)

**Pull request merged**